packages/fluent-theme/src/components/preChatActivity/private/RecolorMonochromeImage.module.css

```
@@ -0,0 +1,9 @@
+:global(.webchat-fluent) .pre-chat-message-activity__recolor-monochrome-image {
+  background-color: var(--webchat-colorNeutralForeground4, #707070);
+}
+
+@media (prefers-color-scheme: dark) {
+  :global(.webchat-fluent) .pre-chat-message-activity__recolor-monochrome-image {
+    background-color: var(--webchat-colorNeutralForeground4, #999);
```

```suggestion
    background-color: var(--webchat-colorNeutralForeground4);
```
We have different value `#707070`

---

Yea, thanks for double checking, this is dark theme.

---

Are you saying passing variables from `<FluentProvider>` is not enough for dark theme?

I imagine there might be a case where we need additional styles from dark theme. We'll probably need to know the dark theme was used from Fluent side somehow. We can't rely on [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) solely as FluentUI doesn't respect nor set the value accordingly (at least last time I checked it didn't).